pkg/reconciler/broker/broker.go

```diff
@@ -183,8 +183,8 @@ func (r *Reconciler) reconcileLegacy(ctx context.Context, b *v1alpha1.Broker) er
 		return nil
 	}
 	b.Status.TriggerChannel = &corev1.ObjectReference{
-		Kind:       triggerChan.Kind,
-		APIVersion: triggerChan.APIVersion,
+		Kind:       "Channel",
```

Comment:
but these could be anything right? `InMemoryChannel`... etc

Reply:
No, this is the legacy path (soon to be ripped out, so it's only channels for realzies)